services/src/messaging/src/main/java/org/openkilda/messaging/model/Switch.java

```diff
@@ -53,7 +52,7 @@ public Switch(
         this.datapath = datapath;
         this.ipAddress = ipAddress;
         this.features = ImmutableSet.copyOf(features);
-        this.ports = ImmutableList.copyOf(ports);
+        this.ports = ports;
```

Comment:
Switch is a value object and must be immutable. Please use copy builder to clone and modify it.